man/rgbasm-old.5

```diff
@@ -0,0 +1,333 @@
+.Sh DESCRIPTION
+This is the list of features that have been removed from the
+.Xr rgbasm 5
+assembly language over its decades of evolution, along with their modern alternatives.
+Its goal is to be a reference for backwards incompatibility, when upgrading an old assembly codebase to work with the latest RGBDS release.
+It does
+.Em not
+attempt to list syntax bugs that were fixed, nor new reserved keywords that may conflict with old identifiers.
+.Sh REMOVED
+.Ss Global labels without colons
+Deprecated in 0.4.0, removed in 0.5.0.
+.Pp
+Labels used to be definable with just a name, but had to be typed in column 1.
+.Pp
+Instead, use explicit colons; for example,
+.Ql Label:
+or exported
+.Ql Label:: .
+.Ss Automatic LD to LDH conversion (rgbasm -l)
+Deprecated in 0.7.0, removed in 0.8.0.
+.Pp
+.Xr rgbasm 1
+used to automatically treat
+.Ql LD
+as
+.Ql LDH
+if the address was known to be in the
+.Ad $FF00-$FFFF
+range, with the
+.Fl L
+flag to opt out.
+.Xr rgbasm 1
+0.6.0 added a
+.Fl l
+flag to opt in instead.
+.Pp
+Instead, use
+.Ql LDH ,
+and remove the
+.Fl L
+and
+.Fl l
+flags from
+.Xr rgbasm 1 .
+.Ss Automatic NOP after HALT (rgbasm -H)
+Deprecated in 0.7.0, removed in 0.8.0.
+.Pp
+.Xr rgbasm 1
+used to automatically insert a
+.Ql NOP
+after
+.Ql HALT ,
+with the
+.Fl h
+flag to opt out.
+.Xr rgbasm 1
+0.6.0 added a
+.Fl H
+flag to opt in instead.
+.Pp
+Instead, use an explicit
+.Ql NOP
+after
+.Ql HALT ,
+and remove the
+.Fl h
+and
+.Fl H
+flags from
+.Xr rgbasm 1 .
+.Ss IMPORT and XREF
+Removed in 0.4.0.
+.Pp
+Symbols are now automatically resolved if they were exported from elsewhere.
+.Pp
+Instead, just remove these directives.
+.Ss __FILE__ and __LINE__
+Deprecated in 0.6.0, removed in 0.7.0.
+.Pp
+Instead, use
+.Ic WARN
+or
+.Ic FAIL
+to print a complete trace of filenames and line numbers.
+.Ss _PI
+Deprecated in 0.5.0, removed in 0.6.0.
+.Pp
+Instead, use
+.Ql DEF _PI EQU 3.141592653 .
+.Ss '\e,' in strings in macro arguments
+Deprecated in 0.5.0, removed in 0.7.0.
+.Pp
+Macro arguments now handle quoted strings and parenthesized expressions as single arguments, so commas inside them are not argument separators and do not need escaping.
+.Pp
+Instead, just use commas without backslashes.
+.Ss '*' comments
+Deprecated in 0.4.1, removed in 0.5.0.
+.Pp
+Instead, use
+.Ql \&;
+comments.
+.Ss Nested macro definitions
+Removed in 0.4.2.
+.Pp
+Instead, put the nested macro definition inside a quoted string (making sure that none of its lines start with
+.Ic ENDM ) ,
+then interpolate that string.
+For example:
+.Bd -literal -offset indent
+MACRO outer
+    DEF definition EQUS """
+        MACRO inner
+            println (\1) - (\\1)
+        \enENDM"""
+    {definition}
+    PURGE definition
+ENDM
+    outer 10
+    inner 3 ; prints 7
+.Ed
+.Ss Negative DS
+Removed in 0.3.2.
+.Pp
+This was used to "rewind" the value of
+.Ic @
+in RAM sections, allowing labeled space allocations to overlap.
+.Pp
+Instead, use
+.Ic UNION .
```

Comment:
Could this be considered as `REPLACED`?

Reply:
IMO no, since transforming usage of negative `DS` to use `UNION` is not easily automatable.